README.md

```diff
@@ -0,0 +1,15 @@
+# Antiplagiarism
+
+Antiplagiarism is a code analysis tool for Visual Studio that finds in the solution the fragments of code that have been copied from another solution. 
+The tool was designed to be used along with the Acumatica ERP application. However, the tool can be used with any other solutions as an instrument to find code duplication.
+The functionality of the tool is similar to the Code Clone Detection functionality, which is available in the Visual Studio Enterprise. (For details about this functionality, see https://docs.microsoft.com/en-us/previous-versions/hh205279(v=vs.140) .)
+
+To work with the copied code fragments that the tool has found, you can open the Antiplagiarism window from the **Tools** main menu of Visual Studio. 
+You can sort the results and open the source file by double clicking the file name in the **Location** column of the table with the results. 
```

Comment:
@k-s-popova actually there are two "location" columns - "Source Location" and "Reference Location" which allow navigation to files of source and reference solutions. Should we write about it in the ReadMe?

Reply:
I would stay with "Source Location". Changed the file.